### ISSUE TITLE
WordPress 5.6 issue: wp_editor meta fields cause an unsaved data Javascript alert

### ISSUE THREAD

**Issue by jonathanstegall** [WordPress/classic-editor#148]
When the Classic Editor is enabled, adding a meta box field with `wp_editor` causes a JavaScript alert when leaving a post edit screen, even when there are no unsaved changes. This appears to be a new issue in WordPress 5.6, as far as I can tell. The issue does not happen with other types of meta fields, and it also doesn't happen if the block editor is enabled instead of Classic Editor.

I've tested this with two plugins so far, each running without any other plugins enabled:

1. [CMB2 (using the wysiwyg type field)](https://wordpress.org/plugins/cmb2/)
2. [Advanced Post Excerpt (using it as it comes installed)](https://wordpress.org/plugins/advanced-post-excerpt/)

I've been able to reproduce this with a clean WordPress install of 5.6 with the default theme.

**Comment by azaozz** [comment]
Can reproduce but only after typing something then deleting it, or after switching to the Text editor tab then back to Visual. The same happens in the main editor, and (I think) has been the case for quite a while.

BTW this is a problem in WordPress, not in the Classic Editor plugin. This plugin only enabled the old Edit Post screen and the old editor. They are still part of WP :) 

It may be possible to fix this, but it's not trivial. If there's a patch, please open a ticket on core's Trac: https://login.wordpress.org/?redirect_to=https://core.trac.wordpress.org/newticket.

**Comment by jonathanstegall** [comment]
Someone had already made a ticket here: https://core.trac.wordpress.org/ticket/52038 so I added a comment to it. I'm not sure why the block editor made the issue go away, but clearly you're right that it's not in this plugin. I'll go ahead and close this.
